packager/mpd/base/adaptation_set.h

```diff
@@ -176,6 +176,13 @@ class AdaptationSet {
   /// @return true if it is a video AdaptationSet.
   bool IsVideo() const;
 
+  /// @return codec.
+  std::string codec() const { return codec_; }
```

Comment:
Use const reference to avoid unnecessary copying:

const std::string& codec() const { return codec_; }